MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -514,14 +514,16 @@ private StreamInfo BuildAudioItem(MediaSourceInfo item, AudioOptions options)
 
         private static List<TranscodeReason> GetTranscodeReasonsFromDirectPlayProfile(MediaSourceInfo item, MediaStream videoStream, MediaStream audioStream, IEnumerable<DirectPlayProfile> directPlayProfiles)
         {
+            var mediaType = videoStream != null ? DlnaProfileType.Video : DlnaProfileType.Audio;
```

Comment:
```suggestion
            var mediaType = videoStream == null ? DlnaProfileType.Audio : DlnaProfileType.Video;
```
Test for true, not for false

Reply:
Me: `has a video?` then use `Video` profile.
You: `has no video?` then use `Audio` profile.
`true` depends on the point of view.

Do you want to minimize negative comparisons?